### DISCUSSION TITLE
Issues最近咋label：等待反馈者这么多

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3442] [讨论]
？如题

**Comment by allMagicNB** [comment]
很多小白交了反馈又不看GitHub呗
信息都不交全，怎么处理呢？

**Comment by Deep-Dark-Forest** [comment]
因为最近要开学了很多人交了issue就没时间看github了

**Comment by ZS2147483647** [comment]
建议改成:等待开发者（需要开发者进一步给出反馈）

**Reply by allMagicNB** [reply]
那不就是待处理

**Reply by ZS2147483647** [reply]
不是啊，待处理是一步到位的

**Reply by Deep-Dark-Forest** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/131975043/c09e5120-db18-484f-af7e-517ded976a91)
lm：我等待我自己？

**Reply by Chiloven945** [reply]
待处理的意思相当于等待开发者确认了，不是开发者把这个加todo list里面等待转变为WIP
